src/status_im2/contexts/chat/photo_selector/view.cljs

```diff
@@ -77,10 +79,14 @@
   [chat-id]
   (rf/dispatch [:chat.ui/camera-roll-get-photos 20])
   (let [selected-images (keys (rf/sub [:chats/sending-image]))]
-    (when selected-images
-      (reset! selected (vec selected-images)))
     [:f>
      (fn []
+       (rn/use-effect-once
+        #(do
```

Comment:
https://github.com/bbatsov/clojure-style-guide#no-multiple-forms-in-function-literals